src/components/Pages/PlaylistItemPage.js

```diff
@@ -20,26 +19,17 @@ const COLLECTION_TYPES_BY_ROUTING = {
 const PlaylistItemPage = ({ t }) => {
   const { id, routeType, tab } = useParams();
 
-  const unit     = useSelector(state => selectors.getDenormContentUnit(state.mdb, id));
+  const unit     = useSelector(state => selectors.getDenormContentUnit(state.mdb, id), shallowEqual);
   const wip      = useSelector(state => selectors.getWip(state.mdb).units[id]);
   const err      = useSelector(state => selectors.getErrors(state.mdb).units[id]);
-  const language = useSelector(state => settings.getLanguage(state.settings));
-
-  //fix bug with unit without collection
-  const [needToFetch, setNeedToFetch] = useState();
 
   const dispatch = useDispatch();
 
   useEffect(() => {
-    setNeedToFetch(!unit || Object.keys(unit.collections).length === 0);
-  }, [id, language, unit]);
-
-  useEffect(() => {
-    if (!wip && !err && needToFetch) {
+    if (!wip && !err && !unit) {
       dispatch(actions.fetchUnit(id));
-      setNeedToFetch(false);
     }
-  }, [dispatch, err, id, wip, needToFetch]);
```

Comment:
@wissog why you remove it?

Reply:
because it is not needed when you remove Object.keys(unit.collections).length === 0)
we just check !unit

Reply:
its fix bug nosourcesif you enter from likutim page.

Reply:
YH units don't have collections and constantly being loaded... so i removed that condition
 what is needed from likutim page?

Reply:
can u give an example? i don't see any problem in likutim